site/content/docs/cli/constructs-reference.md

```diff
@@ -62,22 +62,25 @@ dedicated docs on checkMatch and testMatch](/docs/cli/using-check-test-match/)
 
 ## `Check`
 
-The CLI currently supports two Check types: API Checks and Browser Checks. All checks share the following common properties
-derived from the abstract class `Check`.
-
-- `name`: A friendly name for your Check.
-- `frequency`: How often to run your Check in minutes, i.e. `Frequency.EVERY_1H` for every hour.
-- `locations`: An array of location codes where to run your Checks, i.e. `['us-east-1', 'eu-west-1']`.
-- `privateLocations`: an array of [Private Locations](https://www.checklyhq.com/docs/private-locations/) slugs, i.e. `['datacenter-east-1']`.
-- `activated`: A boolean value if your Check is activated or not.
-- `muted`: A boolean value if alert notifications from your Check are muted, i.e. not sent out.
-- `group`: The `CheckGroup` object that this check is part of.
-- `alertChannels`: An array of `AlertChannel` objects to which to send alert notifications.
-- `tags`: An array of tags to help you organize your Checks, i.e. `['product', 'api']`
-- `runtimeId`: The ID of which [runtime](https://www.checklyhq.com/docs/runtimes/specs/) to use for this Check.
-- `testOnly`: A boolean determining if the Check is available only when `test` runs and not included when `deploy` is executed.
-- `retryStrategy`: A [RetryStrategy](#retrystrategy) object configuring [retries](/docs/retries-and-alerting/) for failed check runs.
-- `doubleCheck`: (deprecated) A boolean value if Checkly should double check on failure. This option is deprecated in favor of `retryStrategy`.
+The CLI currently supports three Check types: API, Browser and Heartbeat Checks.
+
+These Check types share properties derived from the abstract class `Check`.
+
+| Property           | Description                                                                                                                    | Supported in                                 |
+|--------------------|--------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------|
+| `name`             | A friendly name for your Check.                                                                                                | `ApiCheck`, `BrowserCheck`, `HeartbeatCheck` |
+| `frequency`        | How often to run your Check in minutes, i.e. `Frequency.EVERY_1H` for every hour.                                              | `ApiCheck`, `BrowserCheck`                   |
```

Comment:
How about we use `API`, `Browser` and `Heartbeat` in the table and mention the whole name above or below to make it less crowded?
Otherwise, how about have each of these in a different color to be able to quickly find what you're looking for?

Reply:
@shiini2 Oh noooooooo. I turned on auto-merge. Colors is too much for me right now (bc it's all markdown), but I'm up  for shortening it with a follow PR.